numba/tests/test_record_dtype.py

```diff
@@ -251,6 +251,35 @@ def get_charseq_tuple(ary, i):
     return ary[i].m, ary[i].n
 
 
+def get_field1(rec):
+    fs = ('e', 'f')
+    f = fs[1]
+    return rec[f]
+
+
+def get_field2(rec):
+    fs = ('e', 'f')
+    out = 0
+    for f in literal_unroll(fs):
+        out += rec[f]
+    return out
+
+
+fs = ('e', 'f')
```

Comment:
Same comment about globals all-caps/underscore. Also, this ought to go near the top of file.

Reply:
fixed